randovania/games/dread/json_data/Burenia.txt

```diff
@@ -1483,63 +1489,88 @@ Extra - polygon: [[-900.0, -7000.0], [-5000.0, -7000.0], [-5000.0, -10700.0], [-
-  > Event - Shoot Blob from alcove (db_reg_aq_003)
-      Shoot Beam
-  > Event - Pull Lower-Right Grapple Block (grapplepulloff1x2_003)
-      Grapple Beam and After Burenia - db_reg_aq_003
+      All of the following:
+          Morph Ball and After Burenia - Destroy Gravity Suit Floor
+          Any of the following:
+              After Burenia - Prepare Speedboost in Save Station or Use Spin Boost
+              Flash Shift and Gravity Suit
+  > Event - Main Hub Bottom Blob, Left
+      Lay Bomb or Shoot Beam
+  > Event - Main Hub Bottom Grapple Block
+      Grapple Beam and After Burenia - Main Hub Bottom Blob
+  > Ledge above Grapple Block
+      Flash Shift and Gravity Suit and After Burenia - Main Hub Bottom Blob
+  > Water Space Jump Platform
+      After Burenia - Main Hub Bottom Blob
 
 > Ledge above Grapple Block; Heals? False
   * Layers: default
   > Door to Energy Recharge South
       All of the following:
           Gravity Suit
-          Any of the following:
-              Simple IBJ or Use Spin Boost
-              Flash Shift and Walljump (Beginner)
+          Flash Shift or Simple IBJ or Use Spin Boost
   > Door to Storm Missile Gate Room
-      Morph Ball and After Burenia - grapplepulloff1x2_003
-  > Alcove across grapple block
-      After Burenia - db_reg_aq_003 and Use Spin Boost
-  > Event - Shoot Blob from right (db_reg_aq_003)
+      Morph Ball and After Burenia - Destroy Gravity Suit Floor
+  > Alcove Across Grapple Block
+      All of the following:
+          After Burenia - Main Hub Bottom Blob
+          Gravity Suit or Use Spin Boost
+  > Event - Main Hub Bottom Blob, Right
       Shoot Diffusion or Wave
   > Water Space Jump Platform
       Trivial
 
-> Event - Shoot Blob from above (db_reg_aq_003); Heals? False
+> Event - Main Hub Bottom Blob, Above; Heals? False
   * Layers: default
-  * Event Burenia - db_reg_aq_003
+  * Event Burenia - Main Hub Bottom Blob
   > Door to Energy Recharge South
       Trivial
 
-> Event - Shoot Blob from right (db_reg_aq_003); Heals? False
+> Event - Main Hub Bottom Blob, Right; Heals? False
   * Layers: default
-  * Event Burenia - db_reg_aq_003
+  * Event Burenia - Main Hub Bottom Blob
   > Ledge above Grapple Block
       Trivial
 
 > Water Space Jump Platform; Heals? False
   * Layers: default
-  > Door to Save Station South Access (doorpowerpower_023)
+  > Door to Save Station South Access (Lower)
       Horizontal Water Movement
+  > Door to Storm Missile Gate Room
+      All of the following:
+          Gravity Suit and Morph Ball and After Burenia - Destroy Gravity Suit Floor
+          Speed Booster or Single-wall Wall Jump (Beginner) or Simple IBJ or Use Spin Boost
   > Tile Group (SCREWATTACK)
       Trivial
+  > Alcove Across Grapple Block
+      All of the following:
+          Gravity Suit and After Burenia - Main Hub Bottom Blob
+          Flash Shift or Simple IBJ or Use Spin Boost
```

Comment:
It´s possible to jump onto the ledge trivial. It´s pretty tight though. Would say Movement (Beginner)?

Reply:
I can record one

Reply:
This will be added later if it is deemed possible as a movement trick